lib/matobjplist.gi

```diff
@@ -19,7 +19,7 @@
 ############################################################################
 
 BindGlobal( "MakeIsPlistVectorRep",
```

Comment:
Maybe add a comment which explains `check`?

Something along these lines (feel free to rewrite this, or to ignore this suggestion)
```suggestion
# Construct a new vector in the filter IsPlistVectorRep with the given
# basedomain and entries copied form the given list.
# If `check` is set to true *and* ValueOption( "check" ) is true, then
# it is checked that the list entries are in the basedomain.
# So whenever it is known that the input is guaranteed to be in the basedomain,
# pass false for check to avoid these (potentially costly) consistency checksl
BindGlobal( "MakeIsPlistVectorRep",
```

Reply:
O.k.